api4/user.go

```diff
@@ -15,6 +15,7 @@ import (
 	"github.com/mattermost/mattermost-server/mlog"
 	"github.com/mattermost/mattermost-server/model"
 	"github.com/mattermost/mattermost-server/store"
+	"github.com/mattermost/mattermost-server/web"
```

Comment:
OK, I changed my mind about the changes you made that were for things reported by `go fmt` but not by `make check-style`. In particular because of the extensive changes to this file. When I thought it was just some small spacing changes in a couple of files, I didn't think much of it, but there are a lot of lines changed in this file, from a commit (0312a3a581845abdce9e9b03af46f55679f0bd4f) which said it was because of fixing things reported by `check-style` but probably wasn't since `make check-style` passed w/o any errors on the develop branch.

Anyway the result is lets not make unnecessary changes to this code base.

Reply:
Reverted.